src/viewport.js

```diff
@@ -107,6 +107,7 @@ $.Viewport = function( options ) {
         minZoomLevel:       $.DEFAULT_SETTINGS.minZoomLevel,
         maxZoomLevel:       $.DEFAULT_SETTINGS.maxZoomLevel,
         degrees:            $.DEFAULT_SETTINGS.degrees,
+        flipped:            $.DEFAULT_SETTINGS.flipped,
```

Comment:
When the viewer creates the viewport, you should pass the viewer's `flipped` (which came from the options) into the viewport.

Reply:
hum :S viewer has no flipped state now. It is only stored on viewport objects to avoid ambiguity. Do you think it should be stored on viewer to?